server.js

```diff
@@ -20,11 +20,11 @@ app.get('/', (req, res) => {
 })
 
 app.post('/', (req, res) => {
-  const { title, description, mod, upvotes} = req.body;
-  pool.query(`INSERT INTO turingtwocents (title, description, mod, upvotes) VALUES (${title}, ${description}, ${mod}, ${upvotes})`,
+  const { title, description, mod, upvotes, date} = req.body;
+  pool.query(`INSERT INTO turingtwocents(title, description, mod, upvotes, date) VALUES ('${title}', '${description}', ${mod}, ${upvotes}, '${date}')`,
```

Comment:
So these additions of commas are the only things that were keeping our values from being inputted? I keep seeing examples of using the "$1, $2" examples. This clearly works but it would be interesting to try this other way as well.